modules/augment/include/opencv2/augment/flip.hpp

```diff
@@ -35,6 +35,29 @@ namespace augment {
     };
 
 
+
+    class CV_EXPORTS_W FlipVertical : public Transform
+    {
+    public:
+        /* @brief Constructor
+           @param _probability probability that this transformation will be applied to each image
+        */
+        CV_WRAP FlipVertical(const Scalar& _probability);
+
+        /* @brief Apply the vertical flipping to a single image
+           @param _src Input image to be flipped
+           @param _dst Output (flipped) image
+        */
+        CV_WRAP void image(InputArray _src, OutputArray _dst);
+
+        /* @brief Apply the flipping for a single point
+           @param _src Input point to be flipped
+        */
+        virtual Point2d point(InputArray image, Point2d& src);
```

Comment:
The same thing. Just `apply`.

Also please use `const Point2f&` instead of `Point2f&`

Reply:
Ok will do it